src/languages/php.md

```diff
@@ -233,11 +231,10 @@ Another example is to set the timezone of the PHP runtime (though, the timezone
 date.timezone = "Europe/Paris"
 ```
 
-After pushing your file, you can check that the custom PHP configuration
-has been added to your environment by logging in via SSH and type:
+After pushing your file, you can check that the custom PHP configuration has been added to your environment by logging in via SSH and type:
```

Comment:
This bit is weird - I suggest removing it. If you've pushed the right thing, it is _guaranteed_ to be deployed... why would you need to log in to confirm?